libr/socket/socket.c

```diff
@@ -260,80 +260,30 @@ R_API bool r_socket_spawn(RSocket *s, const char *cmd, unsigned int timeout) {
 R_API bool r_socket_connect(RSocket *s, const char *host, const char *port, int proto, unsigned int timeout) {
 	r_return_val_if_fail (s, false);
 #if __WINDOWS__
+#define gai_strerror gai_strerrorA
 	struct sockaddr_in sa;
-	struct hostent *he;
 	WSADATA wsadata;
-	TIMEVAL Timeout;
-	Timeout.tv_sec = timeout;
-	Timeout.tv_usec = 0;
 
 	if (WSAStartup (MAKEWORD (1, 1), &wsadata) == SOCKET_ERROR) {
 		eprintf ("Error creating socket.");
 		return false;
 	}
-	s->fd = socket (AF_INET, SOCK_STREAM, 0);
```

Comment:
why removing all this code? Was it not working? Does this function still work without all this? I don't have a Windows machine at the moment to check.

Reply:
it worked, but it was basically the same as the posix version, so i just removed the common part and used the posix code.
It still works fine, was able to connect to a gdbserver and a rap instance normally